index.Rmd

```diff
@@ -201,12 +201,12 @@ Still Working on it...
 
 # Results
 
-## Top ten most committed crime types
+## Top ten most committed crime types 
 ```{r echo=FALSE}
 kable(top10_Crime_MAN[1:10,]) 
 ```
 
-## The Preference on Time of Top Three Committed Crime Types
+## The Preference on Time of Top Three Committed Crime Types 
```

Comment:
Great job! This graph provides me useful information about the relationship between time and amount of top three committed crime types. I think it could be better if you could add a period of the dataset in the title (2015-2017), add units of data on x- and y-axis that would help readers easier to understand the graph. In addition, please describe the meaning of NA in the footnote, which helps readers better understanding.